CHANGELOG.adoc

```diff
@@ -24,6 +24,7 @@
 
 - https://github.com/eclipse-syson/syson/issues/893[#893] [explorer] Fix _Expand All_ tool in SysON _Explorer_ view
 - https://github.com/eclipse-syson/syson/issues/1398[#1398] [explorer] Fix an issue where a d'n'd of an `Element` in a diagram exposed the `Element` twice in the `ViewUsage` associated to the diagram.
+- https://github.com/eclipse-syson/syson/issues/1407[#1407] [metamodel] `Feature` with `ChainingFeature` should be typped with the type of the last `ChainingFeature`.
```

Comment:
typped => typed

Reply:
`Feature` with `ChainingFeature` should be typped with... => Fix an issue where a `Feature` containing a `ChainingFeature` was not typed with...